DigitalLearningSolutions.Web/Controllers/RegisterController.cs

```diff
@@ -120,6 +162,8 @@ public IActionResult LearnerInformation()
             var centreId = (int)data.RegisterViewModel.Centre;
 
             SetLearnerInformationViewBag(model, centreId);
+            ViewBag.IsCentreSpecific = data.RegisterViewModel.IsCentreSpecific;
+            ViewBag.CentreId = data.RegisterViewModel.Centre.Value;

```

Comment:
I'm not sure why either of these need to go in the viewbag if you are getting their values off a model? 

Also they should be done in the SetLearnerInformationViewBag method if they do. (Or at least pulled into another method since these 2 lines of code are repeated a few times)

Reply:
They only exist in the RegisterViewModel, so for other pages, I've put them into the viewbag if their values are needed. I'll pull them into another method.